src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+/**
+ * TO DO:
+ * Update EXPECTED.TXT
+ */
+
+import duke.command.Command;
+import duke.command.ExecutedCommand;
+import duke.tasks.*;
+import duke.parser.Parser;
+import duke.ui.UI;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    /**
+     * Greeting art
+     */
+    public static final String greetingArt = "placeholder for greeting art\n";
+
+    public static final String goodbyeArt =
```

Comment:
Please follow the correct naming convention for `final` variables. (This applies to lines 21, 23, 47, 52)